code/src/components/Date.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+const Date = ({ date, onDateInputChange, onStepChange, onMinusStepChange }) => {
+	// const { nameInput, onNameInputChange } = props;
+
+	return (
+		<form className="form-container5" tabIndex="4">
+			<label htmlFor="dateInput">
+				<h1>Pick a Date</h1>
+			</label>
+
+			<input
```

Comment:
I really like the fact that you used a date picker as an input! This would also be a great place to have an if/else statement. You could after picking a date show different seats depending on which date they picked or show different movies depending on which date they pick.